test/trip_updates/trip_updates_test.exs

```diff
@@ -9,7 +9,7 @@ defmodule TripUpdates.TripUpdatesTest do
   }
 
   @stop_time_update_with_waiver %StopTimeUpdate{
-    arrival_time: DateTime.new!(~D[2023-07-21], ~T[09:26:08.003], "America/New_York"),
```

Comment:
question: Is it intentional to use `now` instead of a specific date at time? (Below too)